Makefile-macOS.brim

```diff
@@ -1,11 +1,12 @@
 include Makefile
 
 LIBS = -lpcap -lpthread -lz
+libmagic_version = $(shell brew list --versions libmagic | cut -d ' ' -f 2)
 
 nss_libs = /usr/local/opt/nss/lib/libnss.a /usr/local/opt/nss/lib/libpkcs12.a /usr/local/opt/nss/lib/libpkixchecker.a /usr/local/opt/nss/lib/libcrmf.a /usr/local/opt/nss/lib/libnssutil.a /usr/local/opt/nss/lib/libpkixpki.a /usr/local/opt/nss/lib/libnssckfw.a /usr/local/opt/nss/lib/libsoftokn.a /usr/local/opt/nss/lib/libcpputil.a /usr/local/opt/nss/lib/libpkixsystem.a /usr/local/opt/nss/lib/libpkixmodule.a /usr/local/opt/nss/lib/libcerthi.a /usr/local/opt/nss/lib/libnssdbm.a /usr/local/opt/nss/lib/libpkixresults.a /usr/local/opt/nss/lib/libpkixstore.a /usr/local/opt/nss/lib/libfreebl.a /usr/local/opt/nss/lib/libpkixcrlsel.a /usr/local/opt/nss/lib/libpk11wrap.a /usr/local/opt/nss/lib/libsmime.a /usr/local/opt/nss/lib/libcertdb.a /usr/local/opt/nss/lib/libsectool.a /usr/local/opt/nss/lib/libgtest.a /usr/local/opt/nss/lib/libjar.a /usr/local/opt/nss/lib/libpkixparams.a /usr/local/opt/nss/lib/libdbm.a /usr/local/opt/nss/lib/libgtestutil.a /usr/local/opt/nss/lib/libpkixutil.a /usr/local/opt/nss/lib/libnssdev.a /usr/local/opt/nss/lib/libpkcs7.a /usr/local/opt/nss/lib/libnssb.a /usr/local/opt/nss/lib/libcryptohi.a /usr/local/opt/nss/lib/libpkixtop.a /usr/local/opt/nss/lib/libnsspki.a /usr/local/opt/nss/lib/libpkixcertsel.a
 nspr_libs = /usr/local/opt/nspr/lib/libplc4.a /usr/local/opt/nspr/lib/libplds4.a /usr/local/opt/nspr/lib/libnspr4.a
 
 
 suricata: $(suricata_OBJECTS) $(suricata_DEPENDENCIES) $(EXTRA_suricata_DEPENDENCIES)
 	@rm -f suricata$(EXEEXT)
-	$(AM_V_CCLD)$(suricata_LINK) $(suricata_OBJECTS) $(suricata_LDADD) /usr/local/Cellar/jansson/2.13.1/lib/libjansson.a /usr/local/Cellar/bzip2/1.0.8/lib/libbz2.a /usr/local/Cellar/libmagic/5.39/lib/libmagic.a /usr/local/Cellar/lz4/1.9.3/lib/liblz4.a /usr/local/Cellar/libnet/1.2/lib/libnet.a $(nss_libs) $(nspr_libs) /usr/local/Cellar/pcre/8.44/lib/libpcre.a /usr/local/Cellar/libyaml/0.2.5/lib/libyaml.a $(LIBS)
+	$(AM_V_CCLD)$(suricata_LINK) $(suricata_OBJECTS) $(suricata_LDADD) /usr/local/Cellar/jansson/2.13.1/lib/libjansson.a /usr/local/Cellar/bzip2/1.0.8/lib/libbz2.a /usr/local/Cellar/libmagic/$(libmagic_version)/lib/libmagic.a /usr/local/Cellar/lz4/1.9.3/lib/liblz4.a /usr/local/Cellar/libnet/1.2/lib/libnet.a $(nss_libs) $(nspr_libs) /usr/local/Cellar/pcre/8.44/lib/libpcre.a /usr/local/Cellar/libyaml/0.2.5/lib/libyaml.a $(LIBS)
```

Comment:
If we want to simplify all this, we could switch to /usr/local/opt/ paths here. (Unlike /usr/local/Cellar/ paths, they don't include a version number.)

Reply:
@nwt: Ah, fair point. Well, this is already tested out and working, so I can save that tweak for the next pass. Thanks for the re-approval.